Groupie-LFG.toc

```diff
@@ -5,6 +5,7 @@
 ## Author: Gogo, TannerShmoog
 ## SavedVariables: groupielfg_global
 ## SavedVariablesPerCharacter: groupielfg_db
+## X-Curse-Project-ID: 1234567
```

Comment:
@Gogo1951 what's the project ID for Groupie-LFG?

Reply:
https://www.curseforge.com/wow/addons/groupie-lfg

I just put in for an API key, should hear back soon. It's a Google Form so I don't think it's automated.